src/App.js

```diff
@@ -3,7 +3,6 @@ import { Helmet } from 'react-helmet';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import GithubCorner from './components/GithubCorner';
 import PageWrapper from './components/PageWrapper';
-import Footer from './components/Footer';
```

Comment:
Now that footer is not used I think we should remove `Footer.js` as well.

Reply:
Sure